public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/inspector-controls.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * WordPress dependencies
+ */
+import { InspectorControls } from '@wordpress/block-editor';
+import { CheckboxControl, PanelBody, ToggleControl } from '@wordpress/components';
+import { Component, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/*
+ * @todo
+ * make this dynamic
+ * add default schema and options, like speakers?
+ *
+ * commit w/ props mark, mel, corey. look for any others as well
+ */
+
+/**
+ * Render the inspector Controls for the Schedule block.
+ */
+export default class extends Component {
+	render() {
+		const { setAttributes } = this.props;
+		const show_categories = false;
+		const choose_days = true;
+		const choose_tracks = true;
+
+		return (
+			<InspectorControls>
+				<PanelBody title={ __( 'Display Settings', 'wordcamporg' ) } initialOpen={ true }>
+					<ToggleControl
+						label={ __( 'Show categories', 'wordcamporg' ) }
+						checked={ show_categories }
+						onChange={ ( value ) => setAttributes( { show_category: value } ) }
+					/>
+
+					<div className="wordcamp-schedule__control-container">
+						<fieldset>
+							<legend>
+								<ToggleControl
```

Comment:
Technically this is invalid HTML, a legend should only contain "phrasing content", but it _does_ seem to be working. I'll test this more when you've got it actually functional in your next PR.